utils/constants.jsx

```diff
@@ -503,6 +503,213 @@ export const GroupUnreadChannels = {
     DEFAULT_OFF: 'default_off',
 };
 
+export const Permissions = {
+    invite_user: {
+        code: 'invite_user',
```

Comment:
Also, how does this relate to the list of permissions constants in redux?

Reply:
We haven't a good reason to use `code` instead of `id`, probably the reason behind this is that ids are stored in the database and normally have some 26 random characters string, and in this case is a code that identify the permission. But anyway, `id` makes sense too and probably is more homogeneous with the rest of the application.

About constants in redux, the constants in redux that list permissions haven't any information about the scope. Actually, I can rethink it a bit because maybe makes more sense other organization now because this constants has changed multiple times during the development.

Reply:
I going to simplify it

Reply:
Ok, now is based on the mattermost-redux constants and now I only add the scope to permissions.